baremaps-geoparquet/src/test/java/org/apache/baremaps/geoparquet/GeoParquetReaderTest.java

```diff
@@ -17,26 +17,51 @@
 
 package org.apache.baremaps.geoparquet;
 
+import static org.junit.jupiter.api.Assertions.*;
+
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.baremaps.testing.TestFiles;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 class GeoParquetReaderTest {
 
   @Test
   void read() throws IOException, URISyntaxException {
-    // URI geoParquet = new
-    // URI("s3a://overturemaps-us-west-2/release/2024-03-12-alpha.0/theme=admins/type=locality_area/*.parquet");
     URI geoParquet = TestFiles.GEOPARQUET.toUri();
+    final boolean isPrintingContent = true;
```

Comment:
I usually don't print content in the unit tests. I believe this was a left over... I don't mind merging the PR, but I think we should remove this in the future.